.github/workflows/e2e.generic.schedule.main.adversarial-invalidsubjects.slsa3.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           # sha256sum generates sha256 hash for all artifacts.
           # base64 -w0 encodes to base64 and outputs on a single line.
           # sha256sum artifact1 artifact2 ... | base64 -w0
-          echo "::set-output name=hashes::something invalid | base64 -w0)"
+          echo "hashes=$(sha256sum somethinginvalid | base64 -w0)" >> $GITHUB_OUTPUT
```

Comment:
space was removed. probably does not matter. just fyi

Reply:
oh yes - this is the one i was least sure about - i think maybe this was meant to be "something invalid" piped to base64 -w actually, since sha256sum won't find "something invalid" or "somethinginvalid"

I think the syntax of this was a mistake before anyway. 

Maybe I'll change to `echo "something invalid" | sha256sum | base64 -w 0" so it's actually a base 64 encoded sha?

Reply:
@ianlewis can you double check this one if you have a moment? the latest commit has the update

Reply:
I think this test was just checking that the subjects input was validated so I don't think we care about piping to sha256sum really.

Maybe just this is ok,
```shell
echo "artifact1  somethinginvalid" | base64 -w0 >> "${GITHUB_OUTPUT}"
```

Reply:
thanks!